content/news/posts/2024/meet-prestashop9-api.md

```diff
@@ -0,0 +1,386 @@
+
+{{% notice warning %}}
+You can disable the forced secured protocol in the configuration but **only** with the debug mode enabled, production mode is strictly secured and you need HTTPS to connect to the API.
+{{% /notice %}}
+
+![PrestaShop 9 API Configuration](/assets/images/2024/05/api/api_ssl_configuration.jpeg)
+
+## Ouath2 terminology
+
+Before I continue with the tutorial, I have included a table below that will help you understand some of the terms used in this article.
+
+![PrestaShop 9 API terminology](/assets/images/2024/05/api/terminology.png)
+
+Now, let's proceed to the next step.
+
+## Create API client
+
+Let's create our first application. You can add as many applications to API as you want. In the real-world scenario, you probably want to create separate applications for each service you want to integrate. You will probably have clients like "My ERP integration", "Some marketing automation tool client", etc. 
+
+To add a new API client, navigate to Advanced Parameters -> Admin API page. Here you can see a list of all API clients, you can add a new one by clicking the "Add new API Client" button.
+
+![PrestaShop 9 API Client](/assets/images/2024/05/api/api_add_new_client1.jpeg)
+
+If you want to add a new client, you must provide information about it. In the form below, you must provide all the necessary information about the client. After that, you can click the "Generate client secret and save" button. This will generate a client secret for you and save the client in the database. **It's important to save the client secret because you won't be able to see it again.**
+
+![PrestaShop 9 Form for adding a new API client](/assets/images/2024/05/api/api_add_new_client2.jpeg)
+<p class="text-center text-muted small">Form for adding a new API client</p>
+
+* **Client name** - the name of the client, it can be anything you want. It's just for your reference.
+* **Client ID** - the ID of the client, it's a unique identifier for the client. It should be written without spaces or special characters.
+* **Description** - a description of the client, it can be anything you want. It's just for your reference.
+* **Lifetime** - the lifetime of the client. It's the time after which the client access token will expire. In seconds.
+* **Enabled** - if the client is enabled or not. If it's not enabled, the client won't be able to use the API.
+* **Scopes** - the scopes the client can use. You can enable multiple scopes. Each time you add a new scope, you will have to adjust client's settings.
+
+{{% notice %}}
+As part of API extensibility, you can add new scopes to the API. This will allow you to create more granular permissions for your clients. You can find [example module here](https://github.com/PrestaShop/example-modules/tree/master/api_module) and in [ps_apiresources](https://github.com/PrestaShop/ps_apiresources) repository.
```

Comment:
```suggestion
As part of API extensibility, you can add new scopes to the API. This will allow you to create more granular permissions for your clients. You can find an [example module here](https://github.com/PrestaShop/example-modules/tree/master/api_module) and in [ps_apiresources](https://github.com/PrestaShop/ps_apiresources) repository.
```